src/common/pipes/event-validation.pipe.ts

```diff
@@ -1,63 +1,50 @@
-import { ConfigService } from '@nestjs/config';
 import { PipeTransform, Injectable, BadRequestException } from '@nestjs/common';
 import { CreateEventDto } from 'src/modules/event/dto/create-event.dto';
-import { getTimezoneDate } from '../utils/pipe.util';
 import { ERROR_MESSAGES } from '../utils/constants.util';
+import {
+  ValidationArguments,
+  ValidatorConstraint,
+  ValidatorConstraintInterface,
+} from 'class-validator';
 
 @Injectable()
 export class DateValidationPipe implements PipeTransform {
-  constructor(private configService: ConfigService) {}
-
   transform(createEventDto: CreateEventDto) {
-    const timeZone = this.configService.get<string>('TIMEZONE');
-    const startDate = getTimezoneDate(
-      timeZone,
-      new Date(createEventDto.startDatetime),
-    );
-    const endDate = getTimezoneDate(
-      timeZone,
-      new Date(createEventDto.endDatetime),
-    );
-    const currentDate = getTimezoneDate(timeZone); // Current date in the specified timezone
+    const eventStartDate = createEventDto.startDatetime.split('T')[0];
+    const eventEndDate = createEventDto.endDatetime.split('T')[0];
 
-    if (startDate <= currentDate) {
+    const startDate = new Date(createEventDto.startDatetime);
```

Comment:
new Date(createEventDto.startDatetime); will give you date from the parameter right? 
Why     const eventStartDate = createEventDto.startDatetime.split('T')[0]; this code is written?

Reply:
@Xitija